content/en/getting_started/synthetics/api_test.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Getting Started with API Tests
 further_reading:
-- link: 'https://learn.datadoghq.com/courses/intro-to-synthetic-tests'
+- link: 'https://learn.datadoghq.com/courses/getting-started-with-synthetic-browser-testing'
   tag: 'Learning Center'
-  text: 'Introduction to Synthetic Tests'
+  text: 'Getting Started with Synthetic Browser Testing'
```

Comment:
Are all these extra changes intentional? I find it weird that in the api tests docs we are referencing browser tests docs 🤔

Reply:
Good catch @luissmg ! I was fixing the out-dated Learning Center link in bulk fashion and didn't see this one was on the API tests page, I can remove it and do another cross check :)